csrankings-f.csv

```diff
@@ -135,6 +135,7 @@ Farhad Ahani Kamangar,University of Texas at Arlington,http://ranger.uta.edu/~ka
 Farhad Kamangar,University of Texas at Arlington,http://ranger.uta.edu/~kamangar,NOSCHOLARPAGE
 Farhad Pourkamali-Anaraki,University of Massachusetts Lowell,http://www.cs.uml.edu/~farhad,NB2B59cAAAAJ
 Farhad Shahrokhi,University of North Texas,http://www.cse.unt.edu/~farhad,dQ6YZ10AAAAJ
+Farhad Shirani,Florida International University,https://www.cis.fiu.edu/faculty-staff/shirani-farhad/,M1Ab5mEAAAAJ

```

Comment:
This faculty member needs two entries, and this one is incorrect. See https://dblp.org/pid/75/9225.html.

Reply:
> If DBLP has multiple entries for this person, all of them need to be listed. Do not update dblp-aliases.csv.